docs/rancher/cloud-provider.md

```diff
@@ -44,6 +44,13 @@ When spinning up an RKE2 cluster using the Harvester node driver, select the `Ha
 
   ![](/img/v1.1/rancher/rke2-cloud-provider.png)
 
+##### Upgrade RKE2
+The cloud provider can be upgraded by upgrading the RKE2 version, you can upgrade the RKE2 cluster via the Rancher UI as follows:
+1. Click **☰ > Cluster Management**.
+1. Find the guest cluster that you want to upgrade and select ⋮ **> Edit Config**.
+1. Select **Kubernetes Version**.
+1. Click **Save**.
```

Comment:
Let's move this to a new section of ## Upgrade Cloud Provider, also need to describe RKE2, RKE1 & K3S clusters respectively, RKE1 & k3s should be identical since they are all installed via catalog apps.